File: MVP_Article/MVP_Article_Frontend/ClientApp/src/app/model/bestellung.ts
import { Empfaenger } from "./empfaenger";
import { Warenkorbeintrag } from "./warenkorbeintrag";

export class Bestellung{
  public Id : number;
  public Warenkorb: Warenkorbeintrag[];
  public EmpfaengerId : number;
  public Empfaenger: Empfaenger;
}

File: MVP_Article/MVP_Article_Frontend/Model/Bestellung.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MVP_Article_Frontend.Model
{
    public class Bestellung : EntityBase
    {
        public List<Warenkorbeintrag> Warenkorb { get; set; }
        public Empfaenger Empfaenger{ get; set; }
        public int EmpfaengerId { get; set; }
    }
}
